gtsam/hybrid/HybridGaussianConditional.h

```diff
@@ -208,8 +208,8 @@ class GTSAM_EXPORT HybridGaussianConditional
    * @return AlgebraicDecisionTree<Key> A decision tree on the discrete keys
    * only, with the leaf values as the error for each assignment.
    */
-  AlgebraicDecisionTree<Key> errorTree(
-      const VectorValues &continuousValues) const;
+  virtual AlgebraicDecisionTree<Key> errorTree(
```

Comment:
virtual is not needed when override. Also wondering why this method is overwritten in the first place? Should it not be identical to HGF version? If not, comment on how and why it is different.

Reply:
Yeah it is not needed. The changes I made when making HybridGaussianConditional inherit from HybridGaussianFactor were sufficient to accommodate this. :)